index.js

```diff
@@ -1,57 +1,115 @@
 import express from 'express';
 import bodyParser from 'body-parser';
 import fetch from 'node-fetch';
+import weaviate from 'weaviate-client';
 
 const app = express();
 
-app.use(bodyParser.json())
+app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
-const URI = 'http://docker40937-typhon-server.th1.proen.cloud'
-const INDEX = 'poster'
+const URI = 'http://docker40937-typhon-server.th1.proen.cloud';
+const INDEX = 'poster';
 // index list: 'pre-test', 'poster'
 
+const weaviateClient = weaviate.client({
+	scheme: 'http',
+	host: '202.151.177.149:81',
```

Comment:
I think you should keep it as credential information in .env file or export as user data on your cloud server

Reply:
Wow! Thanks for helping. I appreciate it 😀

Reply:
@slothPete7773 
If you change them to .env, please provide .env.example too.

Reply:
Okay, I was just confuse how to inform anyone about the inside `.env`